pkg/config/config.go

```diff
@@ -28,6 +28,9 @@ const (
 	// EnvDEBUG is a custom porter parameter that signals that --debug flag has been passed through from the client to the runtime.
 	EnvDEBUG = "PORTER_DEBUG"
 
+	// EnvDIND is a custom porter parameter that signals to enable "Docker-in-Docker" support.
```

Comment:
This isn't the right location to set this flag (pretty much all the files from here down need to be reverted).